data/www/index.html

```diff
@@ -1,37 +1,750 @@
+              <div class="input-group-append">
+                <button class="btn btn-sm btn-outline-dark" type="button" id="btn-gcommand-checksum" data-toggle="button" aria-pressed="false" onclick="WmButtons.ToggleChecksumDiv();"><i class="icon icon-check-square"></i></button>
+                <button class="btn btn-sm btn-dark mr-1" type="button" id="btn-gcommand">Send <i class="icon icon-chevron-right"></i></button>
+              </div>
+            </div>
+            <div id="checksum-gcommand-div" class="collapse m-0 pr-1 text-right">
+              <span class="field-description mr-1">GCode checksum value:</span>
+              <span class="h5"><span class="badge badge-secondary" id="checksum-gcommand-value" style="width:100px;">&nbsp;</span></span>
+            </div>
+            <div id="checksum-gcommand-div" class="input-group input-group-sm collapse my-1">
+              <input type="text" class="form-control" aria-describedby="checksum-gcommand-value" placeholder="GCode checksum value" readonly/>
+              <div class="input-group-append text-center"><span class="input-group-text" id="checksum-gcommand-value" style="width:100px;"></span></div>
+            </div>
+            <label for="btn-gcommand" class="mb-0 field-labels">Console output:</label>
+            <div class="border rounded p-0 mb-1 console-listbox text-left">
+              <ul class="list-group list-group-flush" id="gcommand-console-list"></ul>
+            </div>
+            <div class="row">
+              <div class="col"><button class="btn btn-sm btn-secondary btn-block" type="button" id="btn-clear-console"><i class="icon icon-ban mr-1"></i> Clear list</button></div>
+              <div class="col"><button class="btn btn-sm btn-success btn-block" type="button" id="btn-export-console"><i class="icon icon-angle-double-down mr-1"></i>Save..</button></div>
+            </div>
+          </div>
+        </div>
+        
       </div>
     </div>
-    <input class="input" name="tabs" type="radio" id="tab-2"/>
-    <label class="label" for="tab-2">controls</label>
-    <div class="panel">
-      <div class="panel-content">
-        #controls
+  </div>
+
+
+  <!-- MODAL: About window ========================================================================================= -->
+  <div class="modal fade" id="modal-info" tabindex="-1" role="dialog" aria-hidden="true">
+    <div class="modal-dialog modal-popup" role="document">
+      <div class="modal-content">
+        <div class="modal-header bg-secondary text-light m-0 px-2 py-1">
+          <span class="h5 font-italic mt-1"><i class="icon icon-info mr-2"></i>Info...</span>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        </div>
+        <div class="modal-body p-2 text-left">
+          <span class="h3">E4d@box Marin WebUI 1.0</span>
```

Comment:
Small typo here. Marin should be Marlin.

Reply:
ops! ok correct that in next review. many thanks.